packages/atlas/src/components/_comments/CommentInput/CommentInput.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+
+import { CommentInput, CommentInputProps } from './CommentInput'
+
+export default {
+  title: 'comments/CommentInput',
+  component: CommentInput,
+  args: {
+    cancelButton: false,
```

Comment:
I would add an image URL to `memberAvatarUrl` as default, just to make this story a little prettier and make sure that this props works.